omod/src/main/java/org/openmrs/module/owa/activator/OwaActivator.java

```diff
@@ -163,4 +172,32 @@ public void stopped() {
 		log.info("OWA Module stopped");
 	}
 	
+	/**
+	 * Used to set the login Url Global Property that directs to the Add-on OWA UI if no UI module
+	 * is installed. Searches against the legacy and Reference Application module on installed UI modules.
+	 */
+	private void configureLoginUrl(AppManager appManager) {
+		if (appManager.exists(ADD_ON_MANAGER_APP)) {
```

Comment:
Will this still work if the addon manager is renamed?

Reply:
This decision will have to be carefully thought to consider the impact that this will cause. I think one good way that'd make this easier to rename is have that **ADD_ON_MANAGER_APP** constant kept in a pool/class of constants to increase renaming convenience.